integration-tests/tests/cookies/cookies.ejs

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Cookies</title>
+
+	<%- renderAgent({debug: true}) %>
+</head>
+<body>
+	<h1>COOKIES</h1>
+
+  <iframe src="/cookies/iframe.ejs"></iframe>
```

Comment:
does it matter that the iframe has the same origin? I'd bet it does

Reply:
It does but we don't have a good way of testing third party iframe. I could start another http server on another port or use some third party  external page somehow. As this is a corner case anyways the test only checks if correct cookie string is used in iframe vs non iframe.

Reply:
Having the capability to serve from a second port would be awesome for, e.g., testing CORS scenarios.  Let's throw that on the backlog but I think what you have here is fine for now.